opentracing/recorder.go

```diff
@@ -55,7 +55,11 @@ func (r *Recorder) RecordSpan(sp basictracer.RawSpan) {
 
 	// Record all of the logs. Payloads are thrown out.
 	for _, log := range sp.Logs {
-		r.collectEvent(spanID, appdash.LogWithTimestamp(log.Event, log.Timestamp))
+		if logs, err := MaterializeWithJSON(log.Fields); err != nil {
```

Comment:
I presume that `MaterializeWithJSON` will only be called here, inside `*Recorder`. So I think it should be unexported?